### DISCUSSION TITLE
emailing members who are not registered

### DISCUSSION THREAD

**Discussion by RamTheRam** [liedekef/events-made-easy#142] [Q&A] [answered]
hello and thank you for this great plugin
I have over 100 members and in some cases they forget to register for an event and they need to be reminded.
I am hoping if there is an option to see who has not registered so I could send them a reminder email
I appreciate your help

**Comment by liedekef** [comment, marked as answer]
Go into email management, tab "Event related email", and then the type "Email to people and/or groups registered in EME", after which you can select the option "Exclude people already registered for the selected event(s)". See this screenshot:
![image](https://user-images.githubusercontent.com/75667/230789104-0a2f76da-a79d-437e-aaa4-21c387e85714.png)
Of course select the people/groups/members/memberships as you'd like then too.

**Comment by RamTheRam** [comment]
My friend, you are great.
now, one more thing, could Event admin add attendees from members' list? like admin registering people for event

**Comment by liedekef** [comment]
Go in the EME backend interface, list of events and click on "RSVP" for the event of choice ...

**Reply by RamTheRam** [reply]
many thanks sir
but, that is one form and everything has to be typed.
 my question was, if a list could show of members so I could add them to the event. I am dealing with a lot of older age members who only expect to receive event emails and rarely track. and when they miss the time they complain.
this would help greatly
